src/main/java/org/apache/ibatis/scripting/xmltags/XMLScriptBuilder.java

```diff
@@ -75,7 +74,7 @@ public SqlSource parseScriptNode() {
     return sqlSource;
   }
 
-  List<SqlNode> parseDynamicTags(XNode node) {
+  private MixedSqlNode parseDynamicTags(XNode node) {
     List<SqlNode> contents = new ArrayList<SqlNode>();
```

Comment:
This method is declared as package private on purpose to avoid synthetic method being generated by the compiler.
There should be a warning depending on IDE setting (it was disabled in my Eclipse).
So, could you revert this line, please?
(Actually, `nodeHandlerMap` should be package private for the same reason.)

Reply:
Sorry, I think `protected` is more appropriate than package private.

Reply:
Thank you. I will commit